packages/block-editor/src/components/contrast-checker/index.js

```diff
@@ -82,21 +82,24 @@ function ContrastChecker( {
 				continue;
 			}
 			message =
-				backgroundColorBrightness < colordTextColor.brightness()
-					? sprintf(
-							// translators: %s is a type of text color, e.g., "text color" or "link color".
-							__(
-								'This color combination may be hard for people to read. Try using a darker background color and/or a brighter %s.'
-							),
-							item.description
+				item.context === 'text'
+					? __(
+							'This color combination may be hard for people to read. Try using a darker background color and/or a brighter text color.'
 					  )
-					: sprintf(
-							// translators: %s is a type of text color, e.g., "text color" or "link color".
-							__(
-								'This color combination may be hard for people to read. Try using a brighter background color and/or a darker %s.'
-							),
-							item.description
+					: __(
+							'This color combination may be hard for people to read. Try using a darker background color and/or a brighter link color.'
 					  );
+			if ( backgroundColorBrightness >= colordTextColor.brightness() ) {
+				message =
+					item.context === 'text'
+						? __(
+								'This color combination may be hard for people to read. Try using a brighter background color and/or a darker text color.'
+						  )
+						: __(
+								'This color combination may be hard for people to read. Try using a brighter background color and/or a darker link color.'
+						  );
+			}
+
```

Comment:
`text color` and `link color` were separate strings and they were making translations harder to understand. This changes the strings to be complete sentences that make sense on their own and are easier to translate.